packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -208,6 +208,16 @@ enum AnimationBehavior {
 /// controllers are created in [State.initState] and disposed in
 /// [State.dispose], as described in the previous section.)
 ///
+/// {@tool dartpad}
+/// This example shows how to use [AnimationController] and [SlideTransition] to create an
+/// animated digit like you might find on a digital clock.  New digit values slide
```

Comment:
```suggestion
/// animated digit like you might find on a mechanical digital clock.  New digit values slide
```
Maybe? I wan to say "analog digital", but that definitely isn't right.  When I think of "digital clock", I think of something with a seven-segment display, and this isn't that.

Reply:
Good point. A better analog would be the circular digit counters in old pinball machine or a car's odometer. Will update the text.